test/e2e/volumes.go

```diff
@@ -219,28 +219,22 @@ func testVolumeClient(client *client.Client, config VolumeTestConfig, volume api
 }
 
 var _ = Describe("Volumes", func() {
-	clean := true // If 'false', the test won't clear its namespace (and pods and services) upon completion. Useful for debugging.
+	framework := Framework{BaseName: "volume"}
 
+	// If 'false', the test won't clear its volumes upon completion. Useful for debugging,
+	// note that namespace deletion is handled by delete-namespace flag
+	clean := true
```

Comment:
Please remove it - this is not used now and you can use your new flag instead now.